packages/editor/src/components/inner-blocks/README.md

```diff
@@ -92,6 +92,13 @@ const TEMPLATE = [ [ 'core/columns', {}, [
 
 The previous example creates an InnerBlocks area containing two columns one with an image and the other with a paragraph.
 
+### `templateInsertUpdatesSelection`
```

Comment:
I wonder if this is really needed or if it should always be "false"?

Reply:
I agree for some blocks it may make sense to use a different setting.